classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] Also setup the has-many association with student in the add_student method(a classroom has many students)).

Reply:
Thanks @Deepakdanger